src/components/input-field/input-field.tsx

```diff
@@ -257,10 +257,11 @@ export class InputField {
         };
 
         return [
-            <label class={classList}>
+            <div class={classList}>
                 {this.renderFormattedNumber()}
                 <input
                     class="mdc-text-field__input"
+                    id="tf-outlined"
```

Comment:
No idea. I guess so. It's from MD's templates.

Reply:
We can call it anything, so let's change that to something that seems less mysterious… I'm sure we can think of something 🙂

Reply:
the question is whether we want to follow MD's templates and conventions or not. But maybe in case of this IDs, it doesn't matter from a developer's perspective? Me not know… 🙃 if you want to change it, I have nothing against it.

Reply:
Absolutely! And it's good that you err on the side of caution 👍

In this case, the `id` is just used for associating the label with the input. It isn't used for styling anything (which is good, since id:s shouldn't be used for styling).

I checked the MDC repo just to be sure, and literally the only place `tf-outlined` appears is in that single example in the readme for `mdc-textfield`, so it's safe to change it 🙂